### PR TITLE
feat: Utilize REST for broadcasting Transactions and Websocket for Transaction Tracing

### PR DESCRIPTION
**Pull request by JoseRFelix**

### CODE REVIEW

**Review by jonator — COMMENTED** [review]
Hey for some reason the overrides in the .env file are not working for me. 

Can you try to look into why that is?

**Review by jonator — COMMENTED** [review]
http://143.198.247.31:1317/
http://143.198.247.31:26657/

Try with these endpoints

**Comment by JoseRFelix** [comment]
> Hey for some reason the overrides in the .env file are not working for me.
> Can you try to look into why that is?

Are you referring to a build or doing a transaction?

**Comment by jonator** [comment]
> > Hey for some reason the overrides in the .env file are not working for me.
> > Can you try to look into why that is?
> 
> Are you referring to build or doing a transaction?

The build

**Comment by JoseRFelix** [comment]
@jonator fixed the env issue on my latest commit. I forgot to include the `.env` files before generating

File: packages/web/package.json [hunk 1]
```diff
@@ -13,9 +13,9 @@
     "lint:fix": "prettier --write \"**/*\" && next lint --fix",
     "pre-commit": "lint-staged",
     "generate": "yarn generate:chain-info && yarn generate:wallet-assets && yarn generate:wallet-registry",
-    "generate:chain-info": "ts-node --esm config/generate-chain-infos/index.ts",
-    "generate:wallet-assets": "ts-node --esm config/generate-wallet-assets/index.ts",
-    "generate:wallet-registry": "ts-node --esm config/generate-wallet-registry.ts"
+    "generate:chain-info": "dotenv -e .env -e .env.local -- ts-node --esm config/generate-chain-infos/index.ts",
```
Comment (jonator) [inline packages/web/package.json:16]: Are we sure `dotenv` doesn't pull in all env files by default, without having to specify them?

There could be many varieties:
```
# local env files
.env.local
.env.development.local
.env.test.local
.env.production.local
```

**Pull request merged**